src/base/system.c

```diff
@@ -1,5 +1,6 @@
 /* (c) Magnus Auvinen. See licence.txt in the root of the distribution for more information. */
 /* If you are missing that file, acquire a complete release at teeworlds.com.                */
+/* (c) Boris Bobrov, 2019 */
```

Comment:
why this line :D

Reply:
The license requires me to

Reply:
Could you quote the part where the license requires you to do that?

Reply:
> 2. Altered source versions must be plainly marked as such, and must not be
misrepresented as being the original software.

According to the discussion in #85 and https://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications

Reply:
Thank you for linking. 
If you want to add your name to every source file you modified, i guess you can do that.
But your comment does not indicate that the source has been changed. A lot of people worked on teeworlds, in theory everyone of them could add their name to the source code. So it is not clear if your name is part of the original source code or a modification. 
This is just one file, out of many that got changed by InfClass. If we wanted to be absolutely correct, i think we should create a string like "Modified source for the InfClass mode, link to github...."
something like this, and then add this string to every file that got modified.
There could also be a list of names at the end of the string, indicating who has worked on it.
But currently no one is doing that, the original code also has no list of contributors.

Reply:
> i think we should create a string like "Modified source for the InfClass mode, link to github...."

That's a great idea. Lets use the following line:

> Copyright \<year\> The InfclassR Authors.

Reply:
Edit:
> Copyright \<year\> The InfclassR (https://github.com/yavl/teeworlds-infclassR/) Authors

Reply:
`Modifications Copyright <year> The Infclas.......`

Reply:
Perfect